modules/GPTQ_loader.py

```diff
@@ -36,32 +36,37 @@ def load_quantized(model_name):
         exit()
 
     path_to_model = Path(f'models/{model_name}')
-    if path_to_model.name.lower().startswith('llama-7b'):
-        pt_model = f'llama-7b-{shared.args.gptq_bits}bit.pt'
-    elif path_to_model.name.lower().startswith('llama-13b'):
-        pt_model = f'llama-13b-{shared.args.gptq_bits}bit.pt'
-    elif path_to_model.name.lower().startswith('llama-30b'):
-        pt_model = f'llama-30b-{shared.args.gptq_bits}bit.pt'
-    elif path_to_model.name.lower().startswith('llama-65b'):
-        pt_model = f'llama-65b-{shared.args.gptq_bits}bit.pt'
+    found_pts = list(path_to_model.glob("*.pt"))
+    pt_path = None
+
+    if len(found_pts) == 1:
+        pt_path = found_pts[0]
     else:
-        pt_model = f'{model_name}-{shared.args.gptq_bits}bit.pt'
+        if path_to_model.name.lower().startswith('llama-7b'):
+            pt_model = f'llama-7b-{shared.args.gptq_bits}bit.pt'
+        elif path_to_model.name.lower().startswith('llama-13b'):
+            pt_model = f'llama-13b-{shared.args.gptq_bits}bit.pt'
+        elif path_to_model.name.lower().startswith('llama-30b'):
+            pt_model = f'llama-30b-{shared.args.gptq_bits}bit.pt'
+        elif path_to_model.name.lower().startswith('llama-65b'):
+            pt_model = f'llama-65b-{shared.args.gptq_bits}bit.pt'
+        else:
+            pt_model = f'{model_name}-{shared.args.gptq_bits}bit.pt'
 
-    # Try to find the .pt both in models/ and in the subfolder
-    pt_path = None
-    for path in [Path(p) for p in [f"models/{pt_model}", f"{path_to_model}/{pt_model}"]]:
-        if path.exists():
-            pt_path = path
+        # Try to find the .pt both in models/ and in the subfolder
+        for path in [Path(p) for p in [f"models/{pt_model}", f"{path_to_model}/{pt_model}"]]:
+            if path.exists():
+                pt_path = path
 
     if not pt_path:
         print(f"Could not find {pt_model}, exiting...")
         exit()
 
     # qwopqwop200's offload
     if shared.args.gptq_pre_layer:
-        model = load_quant(str(path_to_model), str(pt_path), shared.args.gptq_bits, shared.args.gptq_pre_layer)
+        model = load_quant(str(path_to_model), str(pt_path), shared.args.gptq_bits, 128, shared.args.gptq_pre_layer)
     else:
```

Comment:
Could these 128 break act-order? GPTQ states "Currently, groupsize and act-order do not work together and you must choose one of them." So I would imagen using 128 when you are not supposed to will cause issues.

Reply:
This function is imported from here: https://github.com/qwopqwop200/GPTQ-for-LLaMa/blob/main/llama_inference.py#L26

Maybe this function hasn't been updated yet to work with act-order?

Reply:
I changed both 128 to -1 and I was able to load act-order. So I guess just make groupsize a parameter as @sgsdxzy said.

Reply:
If act-order becomes the default just have groupsize default to -1 as it already does on GPTQ.

Reply:
That's good to hear! I will make it a parameter.